src/format-error.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLError, versionInfo } from "graphql";
+import * as utilities from "graphql/error";
+import { GraphQLFormattedError } from "graphql/error";
+
+export const formatError = (error: GraphQLError): GraphQLFormattedError => {
```

Comment:
nit: please refactor to function declaration instead of using arrow function expression.